src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+
+  const splitString = sourceString.trim().split(';');
+
+  splitString.forEach((item) => {
+    const splitItem = item.trim().split(':');
```

Comment:
u can use destructuring here
```suggestion
    const [key, value] = item.trim().split(':');
```